onnx/defs/reduction/defs.cc

```diff
@@ -28,6 +28,13 @@ False instead of True.)DOC";
         "Keep the reduced dimension or not, default 1 mean keep reduced dimension.",
         AttributeProto::INT,
         static_cast<int64_t>(1));
+    if (version == 11) {
+      schema.Attr(
+          "exclude_axes",
+          "Keep the reduced dimension or not, default 1 mean keep reduced dimension.",
```

Comment:
The description does not explain how this is different from `keep_dims`

Reply:
Have removed this attribute from the reduction in the latest pull request